tests/test_management_commands.py

```diff
@@ -17,23 +19,124 @@
+        optionally assert the status and action for the retrieved IndexActions
+        :param index_model: the Index model to retrieve actions for
+        :type index_model: django_elastic_migrations.models.Index
+        :param pre_message: description of the action that just was executed
+        :type pre_message: str
+        :param num_to_get: the number of most recent actions to retrieve
+        :type num_to_get: str
+        :param expected_status: the expected status of the IndexAction to assert; by default STATUS_COMPLETE
+        :type expected_status: django_elastic_migrations.models.IndexAction.STATUSES_ALL
+        :param expected_actions: list of action types to assert, in order from oldest to newest
+        :type expected_actions:  [django_elastic_migrations.models.IndexAction.ACTIONS_ALL]
+        :return: IndexActions for the Index
+        :rtype: [django_elastic_migrations.models.IndexAction]
+        """
+        index_actions = index_model.indexaction_set.all().order_by('-pk')[:num_to_get]
+        num_available = index_actions.count()
+        expected_msg = (
+            "{pre_message} "
+            "the {index_name} index should have had {expected_num} {index_action_plural} available, "
+            "but instead, there {was_were} {actual_num}".format(
+                pre_message=pre_message,
+                index_name=index_model.name,
+                expected_num=num_to_get,
+                index_action_plural="IndexAction{}".format(pluralize(num_to_get)),
+                was_were= pluralize(num_available, "was,were"),
+                actual_num=num_available
+            )
+        )
+        self.assertEqual(index_actions.count(), num_to_get, expected_msg)
+
+        # we ordered them in reverse cron to get most recent actions; put them back in cron order
+        index_actions = list(index_actions)
+        index_actions.reverse()
+
+        if expected_status or expected_actions:
+            for num, index_action in enumerate(index_actions, 1):
+                logs = index_action.log[-300:]
+                expected_msg = (
+                    "{pre_message} "
+                    "the {index_name}'s {ordinal} IndexAction was expected "
+                    "to have status {expected_status}, "
+                    "but instead, it was {actual_status}. \n"
+                    "The IndexAction logs ended with: \n{logs}".format(
+                        pre_message=pre_message,
+                        index_name=index_model.name,
+                        ordinal=ordinal(num),
+                        expected_status=expected_status,
+                        actual_status=index_action.status,
+                        logs=logs
+                    )
+                )
+                self.assertEqual(index_action.status, expected_status, expected_msg)
+
+                if expected_actions:
+                    expected_action = expected_actions[num-1]
```

Comment:
Strictly per your will no real need, but some spaces around "-" wouldn't hurt, python pep standard loves those

Reply:
good call, will update

Reply:
updated in 481968bb9